awx/settings/defaults.py

```diff
@@ -73,6 +73,7 @@ def IS_TESTING(argv=None):
 AWX_CONTAINER_GROUP_POD_LAUNCH_RETRIES = 100
 AWX_CONTAINER_GROUP_POD_LAUNCH_RETRY_DELAY = 5
 AWX_CONTAINER_GROUP_DEFAULT_NAMESPACE = os.getenv('MY_POD_NAMESPACE', 'default')
+AWX_CONTAINER_GROUP_POD_LAUNCH_TIMEOUT = "5m"
```

Comment:
let's be consistent with the name of our receptor param

`AWX_CONTAINER_GROUP_POD_PENDING_TIMEOUT` 

also `AWX_CONTAINER_GROUP_POD_LAUNCH_RETRIES` and `AWX_CONTAINER_GROUP_POD_LAUNCH_RETRY_DELAY` appear to no longer be used anywhere in the app..those should probably be cleaned up

Reply:
let's leave a commend above this param explaining what it does, and some examples of time units they can provide

Reply:
I have made the required changes.